### DISCUSSION TITLE
Why can't I see the quit button in the osu! main menu on my phone?

### DISCUSSION THREAD

**Discussion by Rain-0x01-39** [ppy/osu#27645] [Q&A] [answered]
![IMG_20240317_201339](https://github.com/ppy/osu/assets/83620631/5068c016-b159-4f0e-b26b-b79a54278a51)

**Comment by cihe13375** [comment]
https://github.com/ppy/osu/discussions/16546

**Reply by peppy** [reply, marked as answer]
To explain, phone apps never have exit buttons. Use the home or back button as per every other phone app.
